wp-parsely.php

```diff
@@ -227,3 +228,92 @@ function parsely_integrations( $parsely = null ): Integrations {
 
 	return $parsely_integrations;
 }
+
+add_action( 'admin_init', __NAMESPACE__ . '\\parsely_check_data_scheme_updates', 999 );
```

Comment:
I see both scheme and schema being used here. Schema is probably what we want to use?